docs/overview/excel.md

```diff
@@ -23,7 +23,6 @@ To use Office Scripts, you'll need the following.
 
 1. [Excel on the web](https://www.office.com/launch/excel) (other platforms, such as desktop, are not supported).
 1. OneDrive for Business.
-1. Office Scripts [enabled by your administrator](/microsoft-365/admin/manage/manage-office-scripts-settings).
```

Comment:
Do we want to add a note and say that you may not get this feature if your admin disabled it

Reply:
We have this in the Troubleshooting section. I thought the feedback was that we should stress the positive on-by-default nature?

Reply:
No, we just wanted to make sure admins knew that didn't enable something but I think we could have a highlighted Note section that says something like "if you meet these requirements and still don't see the automate tab, your admin may have disabled...." I know it is in the troubleshooting section, but we list similar things here so we should at least add a note